src/buildercore/config.py

```diff
@@ -81,22 +81,27 @@ class ConfigurationError(Exception):
 ROOTLOG.setLevel(logging.DEBUG) # *default* output level for all LOGs
 
 # StreamHandler sends to stderr by default
-H1 = logging.StreamHandler()
-H1.setLevel(logging.INFO) # output level for *this handler*
-H1.setFormatter(CONSOLE_FORMAT)
+CONSOLE_HANDLER = logging.StreamHandler()
+CONSOLE_HANDLER.setLevel(logging.INFO) # output level for *this handler*
+CONSOLE_HANDLER.setFormatter(CONSOLE_FORMAT)
 
 
 # FileHandler sends to a named file
-H2 = logging.FileHandler(LOG_FILE)
+FILE_HANDLER = logging.FileHandler(LOG_FILE)
 _log_level = os.environ.get('LOG_LEVEL_FILE', 'INFO')
-H2.setLevel(getattr(logging, _log_level))
-H2.setFormatter(FORMAT)
+FILE_HANDLER.setLevel(getattr(logging, _log_level))
+FILE_HANDLER.setFormatter(FORMAT)
 
-ROOTLOG.addHandler(H1)
-ROOTLOG.addHandler(H2)
+ROOTLOG.addHandler(CONSOLE_HANDLER)
+ROOTLOG.addHandler(FILE_HANDLER)
 
 LOG = logging.getLogger(__name__)
 logging.getLogger('paramiko.transport').setLevel(logging.ERROR)
+logging.getLogger('botocore.vendored').setLevel(logging.ERROR)
```

Comment:
what is the effect of this? 

I closed a similar PR recently https://github.com/elifesciences/builder/pull/339/files because the effect on the output would have been a bit radical